Gemfile.lock

```diff
@@ -78,7 +77,6 @@ DEPENDENCIES
   jekyll-feed (~> 0.6)
   minima (~> 2.0)
   tzinfo-data
-  wdm (~> 0.1.0)
```

Comment:
@jamietanna @AnnaDodson I confirmed what we thought on this in #18 

If i run `bundle install` in windows it adds `wdm` and if i run it in WSL or using the docker version, it removes `wdm`. repeatably.

So I don't think we can do anything about that at the mo.

Reply:
This is ok for me, I can toggle between running locally or in docker and the Gemfile sorts itself out. We could potentially add a troubleshooting section in the README if this is a problem for some people.